packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -266,13 +266,16 @@ export class UIFormComponent extends React.Component {
 	}
 
 	render() {
+		const { onSubmitEnter, onSubmitLeave, properties } = this.props;
 		const actions = this.props.actions || [
 			{
 				bsStyle: 'primary',
 				label: 'Submit',
 				type: 'submit',
 				widget: 'button',
 				position: 'right',
+				onMouseEnter: onSubmitEnter ? event => onSubmitEnter(event, properties) : undefined,
```

Comment:
to avoid passing writing the ternary with `undefined`
```suggestion
				onMouseEnter: onSubmitEnter && event => onSubmitEnter(event, properties),
```

Reply:
Good catch :+1: